sync.yml

```diff
@@ -15,7 +34,8 @@ spec:
       labels:
         application: scraper-sync
     spec:
-      # Scraper nodes have sheets permission scopes
+      # Scraper nodes have sheets permission scopes, and we need sheets
+      # permissions to write to the spreadsheet.
```

Comment:
This comment is not true any more right? Which reminds me that we may want an issue to update the docs to eliminate special scope handling on k8s node pool creation for spreadsheet writing, etc.

Reply:
This comment is still true because we have not removed any spreadsheet code, only added the website code.

Created #40.